### DISCUSSION TITLE
How to get a block's reference after template insertion?

### DISCUSSION THREAD

**Discussion by enel1jk** [stdword/logseq13-full-house-plugin#45] [Q&A] [answered]
```
- a001
    - b001 
    - b002 ``{ var x = c.currentBlock }``
    - b003
- a002
    - b004 
    - at here

```

i wan't to get the reference of a001's b002 next to b004, how to do please?

**Comment by stdword** [comment]
Hello! That's nice case! It is not possible right now.
I'll add this option in the next release

**Reply by enel1jk** [reply]
great! thank you

**Reply by stdword** [reply, marked as answer]
Release planned for the next week:
<img width="400px" src="https://github.com/user-attachments/assets/ac0d6a63-f9ff-48f6-9f92-a37d2283a8ca"/>

Documentation is [here](https://stdword.github.io/logseq13-full-house-plugin/#/reference__tags_advanced?id=blocks-uuid)

Issue to track: https://github.com/stdword/logseq13-full-house-plugin/issues/46

**Reply by stdword** [reply]
@enel1jk, it's released!

**Reply by enel1jk** [reply]
i tested, it works, thank you!
